node/main-support.ts

```diff
@@ -443,68 +265,19 @@ function getBestStream(metadata) {
  * @param metadata
  */
 function getFileDuration(metadata): number {
-  if (     metadata
-        && metadata.streams
-        && metadata.streams[0]
-        && metadata.streams[0].duration
-  ) {
+  if (metadata?.streams?.[0]?.duration) {
```

Comment:
@cal2195 -- thought you might like this [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) 🚀  finally available in JS 😎

Reply:
I've missed this so much from Kotlin! ❤️